AGENTS.md

```
@@ -17,49 +17,62 @@ For the frontend, we use Angular and Angular Material with the new Angular Mater
 We use default Angular Material components and styles, avoiding custom themes to ensure consistency and ease of updates. We use both light and dark themes.
 Frontend is always running on background, don't attempt to start it.
 
-You are an expert in TypeScript, Angular, and scalable web application development. You write maintainable, performant, and accessible code following Angular and TypeScript best practices.
+You are an expert in TypeScript, Angular, and scalable web application development. You write functional, maintainable, performant, and accessible code following Angular and TypeScript best practices.
 
 ## TypeScript Best Practices
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)